SubscribeIT/ngDesk
# Fixed render category alignment

Closes #340 

Test case 1
Name :- Alignment category page
1. Login to ngdesk.
2. Goto -> kb -> create article and add it to some section.
3. Goto -> kb -> general.
4. Observe the layout.
5. Observe no error.

Screenshot :- 

![Screenshot from 2022-01-12 17-09-51](https://user-images.githubusercontent.com/89504119/149133809-5cde6298-c062-47fe-bfe4-90aafb3f91e0.png)
